gems/smithy/lib/smithy/anvil/client/templates/shapes.erb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# This is generated code!
+
+module <%= namespace %>
+  module Shapes
+    include Smithy::Client::Shapes
+
+<% shapes.each do |shape| -%>
+    <%= shape.name %> = <%= shape.type %>.new(id: "<%= shape.id %>", traits: <%= shape.traits %>)
+<% end -%>
+
+<% shapes_with_members.each do |shape| -%>
+<% shape.members.each do |member| -%>
+<% case shape.type -%>
```

Comment:
Why not use the type that's already in shapes such as `list` or `map`?

Reply:
I'm not entirely sure what you mean by above but assuming that you are referring to the type defined on the shape hash itself. 

I figured it would be easier to just reference the shape class representation here via `.type` since it is set on the Shape class.